sandbox/readaligner_pairhmm_train.py

```diff
@@ -28,6 +27,11 @@ def trusted_str(cov, trusted_cutoff):
 
 
 def main():
+    try:
```

Comment:
OK, this is a bit of a hack, but if you move the import down below parse_args, you'll be fine.